librz/util/subprocess.c

```diff
@@ -1050,6 +1068,14 @@ RZ_API RZ_OWN RzSubprocess *rz_subprocess_start_opt(RZ_NONNULL const RzSubproces
 	}
 	proc->killpipe[0] = proc->killpipe[1] = -1;
 	proc->ret = -1;
+	proc->ret_sem = NULL;
+	if (opt->ret_sem) {
```

Comment:
Shouldn't this be the opposite?

Reply:
d2ad89da523df4a327b12367273e8658a4227f6e removes `opt->ret_sem`